hbase-client/src/main/java/org/apache/hadoop/hbase/client/AdminOverAsyncAdmin.java

```diff
@@ -1057,4 +1051,9 @@ public void updateRSGroupConfig(String groupName, Map<String, String> configurat
       throws IOException {
     get(admin.updateRSGroupConfig(groupName, configuration));
   }
+
+  @Override
+  public List<LogEntry> getLogEntries(LogRequest logRequest) throws IOException {
```

Comment:
Comment on admin interface has implications here. (_This needs an optional parameter to allow constrained clients to limit the size of the returned list of LogEntry._)